tests/features/rhdm/kieserver/rhdm-kieserver.feature

```diff
@@ -31,3 +31,11 @@ Feature: RHDM KIE Server configuration tests
   Scenario: Check rhdm-kieserver extensions
     When container is ready
     Then container log should contain -Dorg.jbpm.server.ext.disabled=true -Dorg.jbpm.ui.server.ext.disabled=true -Dorg.jbpm.case.server.ext.disabled=true
+
+  @wip
+  Scenario: Check JAVA_OPTS_APPEND on s2i build
+    Given s2i build https://github.com/desmax74/rhdm-7-openshift-image from quickstarts/hello-rules/hellorules using rhdm-1419
+      | variable                        | value                                                                                        |
+      | JAVA_OPTS_APPEND                | -Dawesome.java.params                                                                                       |
+      | KIE_SERVER_CONTAINER_DEPLOYMENT | rhdm-kieserver-hellorules=org.openshift.quickstarts:rhdm-kieserver-hellorules:1.7.0-SNAPSHOT |
+    Then container log should contain -Dawesome.java.params
```

Comment:
you just added a echo on the assemble script, that will be printed in the logs. you should add it when maven is called to perform the build.

Reply:
I'm not 100% sure that is correct, is already passed MAVEN_ARGS_APPEND to add args to maven 
and maven is invoked in several steps  
1) package 
2) install file 
3) pull dependencies
where add the JAVA_OPTS_APPEND in the maven invocations ?